jxmapviewer2/src/main/java/org/jxmapviewer/viewer/MapAttributionPosition.java

```diff
@@ -0,0 +1,9 @@
+package org.jxmapviewer.viewer;
+
+public enum  MapAttributionPosition {
+
+    SOUTH_EAST,
```

Comment:
The values should be top/left/bottom/right, imho since it's a location on the screen, not a geo-location. Swing already has an enum for that.

Reply:
+1